### PR TITLE
Divyanshu | Test | Unable to create cash invoice in UK company, getting error "State not found"

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86cvmmpzg
https://app.clickup.com/t/86cvyvevf

### CODE REVIEW

**Review by dilpreetsinghofficial — APPROVED** [review]

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html [hunk 1]
```diff
@@ -1303,49 +1303,62 @@ <h5>
...
-                                            <a  
-                                                href="javascript:void(0);"
-                                                aria-label="trash"
-                                                *ngIf="!isCompany"
-                                                [tooltip]="commonLocaleData?.app_delete"
-                                                (click)="deletePurchaseBill(item?.uniqueName)"
+                                    <div class="d-flex align-items-center">
+                                        <div class="flex-fill select-action">
+                                            <span [ngClass]="{ 'text-light-2': !item.balanceStatus }">
+                                                {{ item.balanceStatus }}
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html]:
```suggestion
                                                {{ item?.balanceStatus }}
```

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html [hunk 2]
```diff
@@ -1303,49 +1303,62 @@ <h5>
...
+                                                <div
+                                                    id="dropdown-basic"
+                                                    *dropdownMenu
+                                                    class="dropdown-menu dropdown-menu-right width-150"
+                                                    role="menu"
+                                                    aria-label="list of action"
+                                                    [ngClass]="{ 'dropup-menu': isDropUp }"
+                                                >
+                                                    <ul class="select-action-dropdown">
+                                                        <li *ngIf="item.balanceStatus !== 'cancel'" (click)="onPerformAction(item, 'paid')">
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html]:
```suggestion
                                                        <li *ngIf="item?.balanceStatus !== 'cancel'" (click)="onPerformAction(item, 'paid')">
```

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html [hunk 3]
```diff
@@ -1549,10 +1562,15 @@ <h5>
                                                     [ngClass]="{ 'dropup-menu': isDropUp }"
                                                 >
                                                     <ul class="select-action-dropdown">
+                                                        <li (click)="onPerformAction(item, 'paid')" *ngIf="selectedVoucher === 'debit note' || selectedVoucher === 'credit note'">
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html:1565]: Make enum file if not then make constant in enum

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html [hunk 4]
```diff
@@ -1549,10 +1562,15 @@ <h5>
...
+                                                            <a class="cursor-pointer" href="javascript:;" aria-label="payment paid">{{
+                                                                commonLocaleData?.app_payment_status?.paid
+                                                            }}</a>
+                                                        </li>
                                                         <li (click)="onPerformAction(item, 'cancel')">
-                                                            <a class="cp" href="javascript:void(0);" aria-label="cancel">{{
+                                                            <a class="cursor-pointer" href="javascript:;" aria-label="cancel">{{
                                                                 commonLocaleData?.app_payment_status?.cancel
-                                                            }}</a>
+                                                            }} - karde</a>
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html]: remove unused code

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html]: Done

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html (hunk 2 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html]: Done

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html (hunk 3 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html:1565]: Not use Enum in HTML

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html (hunk 4 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html]: Done

**Review by dilpreetsinghofficial — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**